test/Frame.spec.jsx

```diff
@@ -317,36 +317,20 @@ describe('The Frame Component', () => {
     div = document.body.appendChild(document.createElement('div'));
     const willUnmount = sinon.spy();
 
-    class Parent extends React.Component {
-      constructor(props) {
-        super(props);
-        this.state = {
-          isClosed: false
-        };
-      }
-
-      render() {
-        if (!this.state.isClosed) {
-          return (
-            <Frame
-              contentWillUnmount={() => {
-                willUnmount();
-              }}
-            >
-              <div className="test-class-1" />
-            </Frame>
-          );
-        }
-        return null;
-      }
-    }
+    const Thing = () => (
+      <Frame contentWillUnmount={willUnmount}>
+        <div className="test-class-1" />
+      </Frame>
+    );
 
-    const wrapper = ReactDOM.render(<Parent />, div);
+    ReactDOM.render(<Thing />, div);
 
-    wrapper.setState({ isClosed: true }, () => {
+    // TODO: act() doesn'tt seem to solve this something weird???
+    setTimeout(() => {
```

Comment:
@Reflex-Gravity not sure why `act()` didn't work but this might give you a clue as to what's ahppening here, `srcdoc` is async and `act` doesn't seem to wait long enough for the iframe to actually mount but doing the old `setTimeout` next tick trick does...